module-cheatsheets/scRNA-seq-cheatsheet.md

```diff
@@ -1,41 +1,33 @@
-> The hyperlinks found in each piece of code will take you to the documentation for further information on the usage of each command. 
+#### The tables below consist of valuable functions or commands that will help you through this module.
+##### Each table represents a different library/tool and its corresponding commands.
+> Please note that these tables are not intended to tell you all the information you need to know about each command.
+>
+> The hyperlinks found in each piece of code will take you to the documentation for further information on the usage of each command.
 
 <div style="page-break-after: always;"></div>
 
 ### Base `R`
 
 Read the Base `R` documentation [**here**](https://www.rdocumentation.org/packages/base/versions/3.5.1)
 
-| Library/Package      | Piece of Code                           | What it's called                     | What it does                                                                                                           |
-|----------------------|-----------------------------------------|-------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------|                                                         
-| Base `R`               | [`source()`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/source)                              | Source          | Reads input from a specified file or URL                                                                    |
-| Base `R`               | [`which()`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/which)                                | Which           | Assesses an object logically and returns true indices of the object that are `TRUE`                                    |
-| Base `R`               | [`t()`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/t)                                        | Transpose       | Returns the transpose of a matrix or data frame                                                             |
-| Base `R`               | [`prcomp()`](https://www.rdocumentation.org/packages/stats/versions/3.5.1/topics/prcomp)                             | Principal Components Analysis  | Executes a principal components analysis on specified matrix or data frame                                               |
-| Base `R`               | [`<-function(x) { <code> }`](http://adv-r.had.co.nz/Functions.html)                       | Create a function               | Creates a function that would take the defined parameters as input and execute the commands within the curly braces  |
+|Library/Package|Piece of Code|What it's called| What it does|
+|---------------|-------------|----------------|-------------|
+| Base `R`| [`rowSums()`](https://www.rdocumentation.org/packages/raster/versions/3.3-13/topics/rowSums)| Row sums|Calculates sums for each row|
```

Comment:
I was debating which documentation to link here for the machine learning cheatsheets revamp as well. The reason being, the documentation at the link you have mentions calculating sums for Raster objects while the link [here](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/colSums) is more general to numeric array and data frames (although there is no specific link for `rowSums` it is included in the documentation at the `colSums` link).

Reply:
I'll use the link you have, I think it's more appropriately what we are referencing here. Good call!